### PR TITLE
Typo corrections on lessons 0 and 1.

### PR DESCRIPTION
**Pull request by ldconejo**

### CODE REVIEW

File: source/modules/lesson00/expectations.rst [hunk 1]
```diff
@@ -5,26 +5,26 @@ Expectations
...
 
-Also, there are usually ungraded activities for the lessons. This is work
-that you can do to help you really understand and apply the concepts. Although
-the work is ungraded, we expect that you will want to do the work!
+Also, most lessons include ungraded activities. This is work
+that you can do to help you improved your understanding and apply the concepts learned. Although
+ungraded, we expect that you will want to do the work!
 
-Since giving demos is an important part of a modern develops toolset, your classroom
+Since giving demos is an important part of a modern devops toolset, your classroom
```
Comment (PythonCHB) [inline source/modules/lesson00/expectations.rst]: developer's ?

File: source/modules/lesson00/grading.rst [hunk 2]
```diff
@@ -3,11 +3,11 @@ Grading
 
 The purpose of grading is to help students and instructors assess a student's
 progress on the class, and to demonstrate that a student is proficient and
-achieved the class objectives.
+has achieved the class objectives.
 
 There are three grades used to rate every assignment:
```
Comment (PythonCHB) [inline source/modules/lesson00/grading.rst]: We ended up developing a "rubric" for course 1 (primarily the online self-paced version)  -- we may want to see if we can be consistent between the classes.

I can't seem to find it anywhere -- you might ask Debalina or Natasha.

Bu tin any case, we gave credit for the code working, but also some for style and "pythonicity" -- I think it's really important to review and (maybe) score on good structure and style s well as simple functionality.

File: source/modules/lesson00/expectations.rst (hunk 1 above)
Comment (ldconejo) [reply source/modules/lesson00/expectations.rst]: Makes sense, I'll change it to "developer's".

File: source/modules/lesson00/grading.rst (hunk 2 above)
Comment (ldconejo) [reply source/modules/lesson00/grading.rst]: I'll add a TODO to this file to discuss the rubric with Andy.

**Pull request merged**